tests/serverpod_test_server/test/encode_with_type_test.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:test/test.dart';
+
+void main() {
+  var protocol = Protocol();
+
+  group('Given a', () {
```

Comment:
Suggestion: Since there is nothing shared between the tests you might as well remove the group and just add the `Given a` directly to the test name.